frontend/src/app/profile/profile.service.ts

```diff
@@ -27,11 +27,11 @@ export interface Profile {
 }
 
 export interface PublicProfile {
-  id: number;
-  first_name: string;
-  last_name: string;
-  pronouns: string;
-  email: string;
+  id: number | null;
+  first_name: string | null;
+  last_name: string | null;
+  pronouns: string | null;
+  email: string | null;
```

Comment:
Should these really be `| null`? It seems to me every `PublicProfile` should have values for each of these fields and it adds complexity everywhere they are used to check for `null`. The only field that seems arguably hideable is `email`.

Reply:
Agree!